packages/db/db-client-types.ts

```diff
@@ -294,7 +294,7 @@ export type ListPsaPinRequestOptions = {
   /**
    * status  to match
    */
-  status?: Array<definitions['pin']['status']>
+  statuses?: Array<definitions['pin']['status']>
```

Comment:
This makes sense, given that the mapping of filter status to DB status is now in the API package. Although the type is now a mix of API filter attributes and DB attributes.

Reply:
I don't feel this represents a problem, the other options look reasonable in line.